lib/config/applyrule-targeted.cpp

```diff
@@ -0,0 +1,262 @@
+/* Icinga 2 | (c) 2022 Icinga GmbH | GPLv2+ */
+
+#include "base/string.hpp"
+#include "config/applyrule.hpp"
+#include "config/expression.hpp"
+#include <utility>
+#include <vector>
+
+using namespace icinga;
+
+/**
+ * @returns All ApplyRules targetting only specific parent objects including the given host. (See AddTargetedRule().)
```

Comment:
Technically, "targetting" is correct, though uncommon and inconsistent with "targeted" (as used elsewhere in this PR). Not only affects this place but also other comments in the PR.

Reply:
🤷‍♂️